dissect/target/plugins/apps/webserver/apache.py

```diff
@@ -187,66 +216,153 @@ class ApachePlugin(WebserverPlugin):
         for log_dir, log_name in itertools.product(self.DEFAULT_LOG_DIRS, self.ACCESS_LOG_NAMES):
-            access_log_paths.update(self.target.fs.path(log_dir).glob(f"{log_name}*"))
+            self.access_paths.update(self.target.fs.path(log_dir).glob(f"*{log_name}*"))
 
         for log_dir, log_name in itertools.product(self.DEFAULT_LOG_DIRS, self.ERROR_LOG_NAMES):
-            error_log_paths.update(self.target.fs.path(log_dir).glob(f"{log_name}*"))
+            self.error_paths.update(self.target.fs.path(log_dir).glob(f"*{log_name}*"))
 
         # Check default Apache configs for CustomLog or ErrorLog directives
         for config in self.DEFAULT_CONFIG_PATHS:
             if (path := self.target.fs.path(config)).exists():
-                for line in path.open("rt"):
-                    line = line.strip()
+                self._process_conf_file(path)
 
-                    if not line or ("CustomLog" not in line and "ErrorLog" not in line):
-                        continue
+        # Check all .conf files inside the server root
+        if self.server_root:
+            for path in self.server_root.rglob("*.conf"):
+                self._process_conf_file(path)
 
-                    if "ErrorLog" in line:
-                        set_to_update = error_log_paths
-                        pattern_to_use = RE_CONFIG_ERRORLOG_DIRECTIVE
-                    else:
-                        set_to_update = access_log_paths
-                        pattern_to_use = RE_CONFIG_CUSTOM_LOG_DIRECTIVE
+        return self.access_paths, self.error_paths
 
-                    match = pattern_to_use.match(line)
-                    if not match:
-                        self.target.log.warning("Unexpected Apache log configuration: %s (%s)", line, path)
+    def _process_conf_file(self, path: Path) -> None:
+        """Process an Apache ``.conf`` file for ``ServerRoot``, ``CustomLog``, ``Include`` and ``OptionalInclude``."""
+
+        for line in path.open("rt"):
+            line = line.strip()
+
+            if not line:
+                continue
+
+            elif "ServerRoot" in line:
+                match = RE_CONFIG_ROOT.match(line)
+                if not match:
+                    self.target.log.warning("Unexpected Apache 'ServerRoot' configuration in %s: '%s'", path, line)
+                    continue
+                directive = match.groupdict()
+                self.server_root = self.target.fs.path(directive["location"])
+
+            elif "CustomLog" in line or "ErrorLog" in line:
+                self._process_conf_line(path, line)
+
+            elif "Include" in line:
+                match = RE_CONFIG_INCLUDE.match(line)
+                if not match:
+                    self.target.log.warning("Unexpected Apache 'Include' configuration in %s: '%s'", path, line)
+                    continue
+
+                directive = match.groupdict()
+
+                if "*" in directive["location"]:
```

Comment:
Why do not use glob directly?

As python glob rely on fnmatch and apache documentation explicitly mark the location as being fnmatch style (https://httpd.apache.org/docs/2.4/en/mod/core.html#include) 
> Shell-style (fnmatch()) wildcard characters can be used in the filename`